app/src/main/kotlin/com/simplemobiletools/notes/pro/activities/SettingsActivity.kt

```diff
@@ -62,16 +63,16 @@ class SettingsActivity : SimpleActivity() {
         setupNotesImport()
         setupEnableAutomaticBackups()
         setupManageAutomaticBackups()
-        updateTextColors(settings_nested_scrollview)
+        updateTextColors(binding.settingsNestedScrollview)
 
         arrayOf(
-            settings_color_customization_section_label,
-            settings_general_settings_label,
-            settings_text_label,
-            settings_startup_label,
-            settings_saving_label,
-            settings_migrating_label,
-            settings_backups_label,
+            binding.settingsColorCustomizationLabel,
```

Comment:
settingsColorCustomizationLabel should be settingsColorCustomizationSectionLabel , had to fix it in other apps too. Test the UI properly please

Reply:
My bad :/
Tested it but didn't notice that item on top